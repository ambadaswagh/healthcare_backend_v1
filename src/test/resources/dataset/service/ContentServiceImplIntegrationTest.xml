<?xml version="1.0" encoding="UTF-8"?>
<dataset>
   <content id  = "100"
			title = "test title"
			content = "VGVzdCBUZXh0"
			page_title = "login user"
			page_keyword = "login page"
			page_description = "Login user"
			access_key = "loginKey"
			created_at = "2017-05-05"
			updated_at = "2017-05-05"
			status = "1"
			base_id = "1" />
    
    		  
</dataset>