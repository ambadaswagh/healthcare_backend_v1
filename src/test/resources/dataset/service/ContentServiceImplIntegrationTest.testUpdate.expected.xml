<?xml version="1.0" encoding="UTF-8"?>
<dataset>

       <content id  = "100"
			title = "test title updated"
			content = "VGVzdCBUZXh0"
			page_title = "login user updated"
		/>    


</dataset>